src/main/java/dev/jbang/util/Util.java

```diff
@@ -217,17 +217,29 @@ public static List<String> explode(String source, Path baseDir, String filepatte
 			// not a pattern thus just as well return path directly
 			results.add(filepattern);
 		} else {
-			// it is a non-url letls try locate it
+			// it is a non-url let's try to locate it
+			final Path bd;
+			final boolean useAbsPath;
+			if (Paths.get(filepattern).isAbsolute()) {
```

Comment:
what is this trying to do?

Reply:
The "it's now also possible to use absolute paths together with globbing." part from the commit message ;-)

For the tests it's easier if you can just do `--sources /full/path/to/some/test/files*` so I made that work. (Absolute paths were already supported so it would be strange not to allow globbing in those paths)

Reply:
this should get tested with C:, D: paths on Windows

Reply:
this should get tested with C:, D: paths on Windows